string parent_model
string parent_link
string child_model
string child_link
string joint_name
---
bool status
